src/components/ExploreMore.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+import React from "react";
+import { Box, Button, Typography } from "@mui/material";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import TitleResponse from "./TitleResponse";
+
+interface ExploreMoreData {
+	headline: string;
+	startupName: string;
+	descriptionShort: string;
+	productLink: string;
+	demoLink: string;
+	descriptionLong: string;
+}
+
+interface ExploreMoreProps {
+	data: ExploreMoreData;
+	onBack: () => void;
+}
+
+const ExploreMore: React.FC<ExploreMoreProps> = ({ data, onBack }) => {
+	return (
+		<Box
+			sx={{
+				width: "100%",
+				maxWidth: "500px",
+				backgroundColor: "var(--color-background-paper)",
+				padding: "48px",
+				borderRadius: "16px",
+				border: `1px solid var(--color-border)`,
+				boxShadow: "var(--box-shadow)",
+				display: "flex",
+				flexDirection: "column",
+				gap: "var(--spacing-medium)",
+				alignItems: "center",
+				fontFamily: "var(--font-family-outfit)",
+			}}
+		>
+			<Box
+				sx={{
+					width: "100%",
+					display: "flex",
+					alignItems: "center",
+					justifyContent: "center",
+					position: "relative",
+				}}
+			>
+				<ArrowBackIosNewIcon
```

Comment:
Using ArrowBackIosNewIcon for the back button is helpful for navigation. I am thinking adding a tooltip here could be a small improvement for accessibility.

Reply:
Maybe we can go back to this later. There is no tooltip in the current design, so I would need to get input from Saleh on that. We can definitely go back to this later